fastplotlib/graphics/_base.py

```diff
@@ -61,3 +61,13 @@ def children(self) -> pygfx.WorldObject:
 
     def update_data(self, data: Any):
         pass
+
+    def __repr__(self):
+        title = "Fastplotlib " + self.__class__.__name__ + "\n"
```

Comment:
returning "fastplotlib" is kinda redundant, how about something like this which is shorter:

```
"user-given-name" <__class__.__name__> Graphic @ <hex-id>
```